packit/utils.py

```diff
@@ -85,7 +86,16 @@ def new_sources(self, sources="", fail=True):
             fail=fail,
         )
 
+    def build(self):
+        return run_command(
+            cmd=f"fedpkg{'-stage' if self.stage else ''} build --nowait",
+            cwd=self.directory,
+            error_message="Submission of build to koji failed.",
+            fail=True
+        )
+
     def init_ticket(self, keytab: str = None):
+        # TODO: this method has nothing to do with fedpkg, pull it out
```

Comment:
Do we need kerberos for other operations? Or is `fedpkg` the only one?

Reply:
bodhi can do kerb as well